napari/_qt/dialogs/_tests/test_qt_plugin_dialog.py

```diff
@@ -29,7 +29,11 @@ def _iter_napari_hub_or_pypi_plugin_info(
         "license": "UNKNOWN",
     }
     for i in range(2):
-        yield PackageMetadata(name=f"test-name-{i}", **base_data), bool(i)
+        yield PackageMetadata(name=f"test-name-{i}", **base_data), bool(i), {
```

Comment:
I believe it is used, but I'm looking into it more.  This only happened with a newer version of napari-animations?

Reply:
I see the same now.  Thank you for reporting this!

Reply:
Do you mean the exception posted in the main comment? I test only `napari-animations`. But it will happen for all plugins that will add new things in a new release because of trying to reimport the already imported packages.

Reply:
Yes, I see that.  Also, and I just checked and this is a problem on the main repo and not introduced with this PR.  My suggestion is to open a new issue and subsequent PR to handle this problem, as this seems to be outside the scope of this PR.  What do we think?

Reply:
I can put in a pop up for now to ask the user to restart until this gets fixed?

Reply:
thats true.  I guess I was thinking it might be caught by a try/except, and we can get out of the situation?  I haven't tried it and maybe I'm missing something?